doc/developer_guide/index.md

```diff
@@ -51,11 +55,34 @@ The `.pixi` directory can be large, so it is advised not to put the source direc
 
 The source code for the Panel project is hosted on [GitHub](https://github.com/holoviz/panel). The first thing you need to do is clone the repository.
 
+::::{tab-set}
+
+:::{tab-item} Contributor
+
 1. Go to [github.com/holoviz/panel](https://github.com/holoviz/panel)
 2. [Fork the repository](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#forking-a-repository)
 3. Run in your terminal: `git clone https://github.com/<Your Username Here>/panel`
 
+:::
+
+:::{tab-item} Core Contributor
+
+As a core contributor you should be able to `git push` directly to the `panel` repository. You thus don't need your own _fork_ of the project.
+
+1. Run in your terminal: `git clone https://github.com/holoviz/panel.git`
+
+:::
+
+::::
+
 The instructions for cloning above created a `panel` directory at your file system location.
+
+Now navigate to the root of the `panel` project:
+
+```bash
+cd panel
```

Comment:
If you can't `cd` into a directory, you are likely unable to contribute :upside_down_face:

Reply:
But for example if you are leading a sprint at a conference some people with minimal technical skills will join to not only contribute, but to learn.

I would argue we should make the developer guide as explicit as possible.